lib/minify.js

```diff
@@ -125,6 +131,9 @@ function minify(files, options) {
             }
             toplevel = options.parse.toplevel;
         }
+        if (quoted_props) {
+            reserve_quoted_keys(toplevel, quoted_props);
+        }
```

Comment:
Creating the reserved prop list for `options.mangle.properties.keep_quoted` would show up in the `--timings` for `parse`?

Might this be broken out and added to the `--timings` for `properties` (and subtracted from `parse`)?